### ISSUE TITLE
ignore(element ...)

### ISSUE THREAD

**Issue by fpillet** [RxSwiftCommunity/RxSwift-Ext#7]
### Name of the operator

`ignore(element ...)`

### Motivation for inclusion

This operator allows ignore one or more specific elements emitted by the source sequence. The idea is to have the operator take a variable number of parameters (all of the element type) or an array.

The only constraint is that elements must conform to the `Equatable` protocol.

### Example of usage

```swift
  [1,2,3,4,5,6,7,8,9,10].ignore(5,6,7).subscribeNext { print($0) }
  // prints all but 5,6,7 items
```

**Comment by icanzilb** [comment]
👍I also use that in my code, definitely useful

**Comment by fpillet** [comment]
Implemented in 6d19ad9
